litmus-portal/frontend/src/components/CustomResultModal/index.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { useState } from "react";
+import { useStyles } from "./styles";
+import { Typography } from "@material-ui/core";
+import ButtonFilled from "../../components/ButtonFilled/index";
+import Modal from "@material-ui/core/Modal";
+import Backdrop from "@material-ui/core/Backdrop";
+import Fade from "@material-ui/core/Fade";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import CheckIcon from "@material-ui/icons/Check";
+
+function createData(
+	name: string,
+	result: JSX.Element,
+	weight: number | number[],
+	points: number
+) {
+	return { name, result, weight, points };
+}
+const result = 9;
+const result1 = 6;
+const result2 = 7;
+const result3 = 8;
+
+interface customModalProps {
+	isOpen: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+	testValue: (number | number[])[];
+}
+
+export default function CustomResultModal(props: customModalProps) {
+	const classes = useStyles();
+	const { isOpen, testValue } = props;
+
+	const rows = [
+		createData(
+			"Node add test",
+			<div>
+				<CheckIcon style={{ color: "#109B67" }} />{" "}
+				<Typography>Pass</Typography>
+			</div>,
+			testValue[0],
+			result
+		),
+		createData(
+			"Config map multi volume test",
+			<div>
+				<CheckIcon style={{ color: "#109B67" }} />{" "}
```

Comment:
Try not to use inline styling in general, and also no colors should be used in HTML as inline css, please change this to theme data

Reply:
I guess, I've updated this in the newer commits.